test/views/payments/OpenPaymentsViewSpec.scala

```diff
@@ -680,11 +689,34 @@ class OpenPaymentsViewSpec extends ViewBaseSpec {
         elementText(Selectors.paymentDueData(17)) shouldBe "due by 5 April 2017"
       }
 
-      "not display a view return link text for the 16th payment" in {
+      "not display a view return link text for the 17th payment" in {
         document.select(Selectors.paymentViewReturnText(17)).size shouldBe 0
       }
     }
 
+    "for the 18th payment" should {
+
+      "render the correct amount for the 18th payment" in {
+        elementText(Selectors.paymentAmount(18)) shouldBe "£1,800"
+      }
+
+      "render the correct amount for the 18th payment amount data attribute" in {
+        elementText(Selectors.paymentAmountData(18)) shouldBe "£1,800"
+      }
+
+      "render the correct due period for the 18th payment" in {
+        elementText(Selectors.paymentDue(18)) shouldBe "due by 5 April 2018"
+      }
+
+      "render the correct due period for the 18th payment period data attribute" in {
+        elementText(Selectors.paymentDueData(18)) shouldBe "due by 5 April 2018"
+      }
+
+      "not display a view return link text for the 18th payment" in {
+        document.select(Selectors.paymentViewReturnText(18)).size shouldBe 0
+      }
```

Comment:
Could you add a test to check the hint text please? i.e. the `further interest due on the protective assessment as this was not paid on time` bit